doc/readme/DEVELOPING.md

```diff
@@ -148,16 +148,16 @@ Then
 
 The option `--leak-check=full` for valgrind might be useful.
 In addition to manually running valgrind on your tests, you can use `ctest` to check multiple tests.
-This requires to be enable in dashboard reports in via `-DBUILD_DASHBOARD_REPORTS=ON`.
+This requires to be enabled in dashboard reports in via `-DBUILD_DASHBOARD_REPORTS=ON`.
 For example
 
     ctest -D ExperimentalMemCheck -R unit-GaussianProcessRegression
 
 #### Adding tests
 We aim to write clear, minimal, yet exhaustive tests of basic building blocks in Shogun.
-Whenever you send us C++ code, we will ask you for a unit test for it.
+Whenever you send us a C++ code, we will ask you for a unit test for it.
 We do test numerical results as compared to reference implementations (e.g. in Python), as well as corner cases, consistency etc.
-Read on [test driven development](https://en.wikipedia.org/wiki/Test-driven_development), and search the web for tips on unit tests, e.g. [googletest's tips](https://github.com/google/googletest/blob/master/googletest/docs/Primer.md).
+Read on [test-driven development](https://en.wikipedia.org/wiki/Test-driven_development), and search the web for tips on unit tests, e.g. [googletest's tips](https://github.com/google/googletest/blob/master/googletest/docs/Primer.md).
```

Comment:
or TDD, as most people seem to call it now

Reply:
Incorporate it.